### DISCUSSION TITLE
commands is not synchronized

### DISCUSSION THREAD

**Discussion by Borzoff** [Rapptz/discord.py#9405] [Q&A]
Hello, I do not synchronize commands, moved the bot to the new discord server, changed guildid, and still do not synchronize 

```python

 MY_GUILD = discord.Object(id=1094568341189972032)

class PBBot(commands.Bot):
    def __init__(self):
        super().__init__(intents=discord.Intents.all(), command_prefix="pb!")

    async def setup_hook(self):
        await self.load_extension(f"cogs.commands")
        await self.tree.sync(guild=MY_GUILD)


bot = PBBot()


async def main():
    await bot.start(os.getenv("TOKEN"))


asyncio.run(main())

**Comment by No767** [comment]
Seems like you are autosyncing your command tree, which is a poor practice. Use [Jishaku](https://jishaku.readthedocs.io/en/latest/index.html)'s sync command (manual sync) or Umbra's sync command instead

**Reply by mikeshardmind** [reply]
This really has nothing to do with the issue

**Comment by ShrootBuck** [comment]
Why're you passing guild as a discord.Object? It should just be the ID, no weird stuff.

**Reply by lightlybakedman** [reply]
There's nothing wrong with using discord.Object. It's not "weird stuff". 

> It should just be the ID

No it shouldn't, the guild kwarg takes a snowflake.

**Reply by ShrootBuck** [reply]
It's not wrong but it's unnecessary

**Reply by lightlybakedman** [reply]
No it isnt. that's the most efficient way. Using get_guild would be unnecessary.

**Comment by lightlybakedman** [comment]
Please show how you're defining your commands.
